content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/retire_service.rb

```diff
@@ -1,5 +1,5 @@
 #
-# Description: This method attempts to retire all of the vms under this top level service
+# Description: Placeholder method for custom automation before retiring sub-resources
```

Comment:
Hi @gmcculloug Thanks for resolving this issue. :-)
Can you remove  "before retiring sub-services" from the description?
I think that part is misleading since retire tasks are created for all of the Services retireable service resources, and are sent to Automate at the same time.